tasklist/client/package.json

```diff
@@ -7,7 +7,7 @@
     "@bpmn-io/element-template-icon-renderer": "0.5.2",
     "@bpmn-io/form-js-carbon-styles": "1.11.2",
     "@bpmn-io/form-js-viewer": "1.11.2",
-    "@camunda/camunda-composite-components": "0.9.0",
+    "@camunda/camunda-composite-components": "0.9.1-rc.2",
```

Comment:
We probably need to update this to patch/minor version before we merge this PR.
Please review it and i will update this before i merge it.